staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/managedfields.go

```diff
@@ -111,6 +111,9 @@ func decodeManagedFields(encodedManagedFields []metav1.ManagedFieldsEntry) (mana
 	managed.fields = make(fieldpath.ManagedFields, len(encodedManagedFields))
 	managed.times = make(map[string]*metav1.Time, len(encodedManagedFields))
 	for _, encodedVersionedSet := range encodedManagedFields {
+		if encodedVersionedSet.FieldsType != "FieldsV1" {
```

Comment:
is this decoding path called during API object decoding?

Reply:
This is performed lazily on creates/updates/patches.